internal/iolimits/iolimits_test.go

```diff
@@ -23,7 +24,7 @@ func TestReadAtMost(t *testing.T) {
 		{bytes.MinRead*5 + 1, bytes.MinRead * 5, false},
 	} {
 		input := make([]byte, c.input)
-		_, err := rand.Read(input)
+		_, err := rng.Read(input)
 		require.NoError(t, err)
 		result, err := ReadAtMost(bytes.NewReader(input), c.limit)
 		if c.shouldSucceed {
```

Comment:
In https://github.com/containers/image/commit/521155ffe6fee9f06349f4a10948367a16aade98, the Makefile had a slight change at line 58, bumping lint from v1.49.0 to v1.51.0.  Is that not needed here?

Reply:
The CI environment we have is running Go 1.18., so the linter update to support Go 1.20 is not necessary.

(I’m not claiming that Go 1.18 is the _right_ Go version to require on this branch, I didn’t investigate that in detail — it’s just a version that happens to be used right now, based on `IMAGE_SUFFIX` in `.cirrus.yml` I think.)

---

It’s quite possible that various backported lint cleanups are not necessary either, they were just making my life easier when backporting (because my development system has `golangci-lint 1.60.2` installed).

Reply:
… ah, perhaps a better way to actually answer the question:

I backported the lint warning cleanup because it was annoying me in local development with a recent linter. I didn’t backport the linter version bump itself because I thought that’s undesirable on a stable branch.

Arguably my local development setup which motivates me to backport unnecessary cleanups is a problem that should be fixed; and strictly speaking, maybe this lint backport should be reverted as unnecessary on the stable branch as well.